src/Microsoft.PowerShell.Commands.Utility/Microsoft.PowerShell.Commands.Utility.csproj

```diff
@@ -2,7 +2,8 @@
   <Import Project="..\..\PowerShell.Common.props" />
   <PropertyGroup>
     <Description>PowerShell's Microsoft.PowerShell.Commands.Utility project</Description>
-    <NoWarn>$(NoWarn);CS1570;CA1416</NoWarn>
+    <!-- CA2213: https://github.com/PowerShell/PowerShell/issues/15803 -->
+    <NoWarn>$(NoWarn);CS1570;CA1416;CA2213</NoWarn>
```

Comment:
Why do we need this if we suppress with pragma in files?

Reply:
I could not suppress with pragma as the error was not tracible to a specific line of code, please see the PR description.